bewcloud.config.sample.ts

```diff
@@ -32,6 +32,8 @@ const config: PartialDeep<Config> = {
   //   from: 'help@bewcloud.com',
   //   host: 'localhost',
   //   port: 465,
+  //   tlsMode: "auto",  // Must be set if port is not 465; may also be "immediate", "starttls" or "none"; "auto" defaults to immediate TLS if port is 465, StartTLS otherwise
```

Comment:
This should match the type JSDoc:


```suggestion
  //  "auto" means "immediate" on port 465, "starttls" otherwise; `null` is legacy behaviour and will be removed in v4; it means "immediate" if port is 465, but opportunistic StartTLS or plain transmission otherwise.
```